runbotics/runbotics-orchestrator-ui/src/src-landing/components/Navbar/Navbar.module.scss

```diff
@@ -21,25 +21,34 @@
     }
 }
 
+.logInBtn {
+    display: none;
+}
+
 @media (max-width: $breakpoint-md) {
     .navLinksExpanded,
     .hideNavLinksExpanded {
         position: absolute;
         top: $HEADER_HEIGHT;
         display: flex;
         flex-direction: column;
+        align-items: center;
         overflow-y: clip;
         transition: all 0.3s;
         border-style: solid;
         border-width: 2px 0;
-        border-color: $palette-background-tertiary;
+        border-color: #F5F5F5;
```

Comment:
this should be added to our theme pallete, also delete instead of comenting out. If we need it back we will use git